daq/ipaddr_test.py

```diff
@@ -60,6 +62,15 @@ def _dhcp_port_toggle_test(self):
         self.host.connect_port(True)
         self._ip_callback = self._next_test
 
+    def _dhcp_lease_wait_test(self):
+        if not self.host.connect_port(False):
+            self.log('disconnect port not enabled')
+            return
+        lease = int(self.lease_time[:-1])
```

Comment:
need check that trailing character is actually 's' -- otherwise very confusing if somebody inadvertently enters "100" instead of "100s"... Need to make that an obvious error.

Reply:
@grafnu I was testing this behaviour. It seems as if when there is no 's' at the end of the lease time, DAQ fails when providing the initial DHCP lease. However, I will make this change.